src/views/SimpleAccount/DepositLists.tsx

```diff
@@ -38,24 +38,26 @@ export const SimpleV4DepositList: React.FC<{ usersAddress: string }> = (props) =
 }
 
 /**
- *
+ * Displays V3 deposits (hidden after loading if empty).
  * @param props
  * @returns
  */
 export const SimpleV3DepositList: React.FC<{ usersAddress: string }> = (props) => {
   const { usersAddress } = props
-  const { data } = useUsersV3PrizePoolBalances(usersAddress, false)
+  const { data, isFetched } = useUsersV3PrizePoolBalances(usersAddress)
   const { t } = useTranslation()
-  return (
-    <div>
-      <CardTitle
-        className='mb-2'
-        title={`V3 ${t('deposits')}`}
-        secondary={`$${data?.totalValueUsd.amountPretty || '0.00'}`}
-      />
-      <V3DepositsList usersAddress={usersAddress} />
-    </div>
-  )
+  if(!isFetched || data.balances.length > 0) {
+    return (
+      <div>
+        <CardTitle
+          className='mb-2'
+          title={`V3 ${t('deposits')}`}
+          secondary={`$${data?.totalValueUsd.amountPretty || '0.00'}`}
+        />
+        {isFetched ? <V3DepositsList usersAddress={usersAddress} balances={data.balances} /> : <LoadingList />}
```

Comment:
`isFetched` needs to be true to get to this state so we don't need this loading state

Reply:
Fixed through d8b2b5b